en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md

```diff
@@ -0,0 +1,352 @@
+- [Pixhawk 6C mini](../flight_controller/pixhawk6c_mini.md) with [PM02 V3](../power_module/holybro_pm02.md)
+
+The approximate maximum size of the FC is: 50x110x22mm
+
+## Additional Accessories
+
+- [GPS F9P (included in Skynode eval. kit)](../gps_compass/rtk_gps_holybro_h-rtk-f9p.md)
+- [GPS M9N (cheaper alternative to F9P)](../gps_compass/rtk_gps_holybro_h-rtk-m8p.md)
+- [Airspeed sensor (included in Skynode eval. kit)](https://www.dualrc.com/parts/airspeed-sensor-sdp33) — recommended for improved safety and performance
+- [Airspeed sensor (cheaper alternative)](https://holybro.com/products/digital-air-speed-sensor?pr_prod_strat=use_description&pr_rec_id=236dfda00&pr_rec_pid=7150470561981&pr_ref_pid=7150472462525&pr_seq=uniform)
+- [Lidar Lightware lw20-c (included in Skynode eval. kit)](../sensor/sfxx_lidar.md) (Optional)
+- [Lidar Seeed Studio PSK-CM8JL65-CC5 (cheaper alternative)](https://www.seeedstudio.com/PSK-CM8JL65-CC5-Infrared-Distance-Measuring-Sensor-p-4028.html)
+- [5V BEC](http://www.mateksys.com/?portfolio=bec12s-pro)
+- [Radio (RC) System](../getting_started/rc_transmitter_receiver.md) of your preference
+- [Servo cable extension cable male 30cm 10 pcs](https://www.getfpv.com/male-to-male-servo-extension-cable-twisted-22awg-jr-style-5-pcs.html)
+- [USB-C extension cable](https://www.digitec.ch/en/s1/product/powerguard-usb-c-usb-c-025-m-usb-cables-22529949?dbq=1&gclid=Cj0KCQjw2cWgBhDYARIsALggUhrh-z-7DSU0wKfLBVa8filkXLQaxUpi7pC0ffQyRzLng8Ph01h2R1gaAp0mEALw_wcB&gclsrc=aw.ds)
+- [3M VHB tape](https://www.amazon.in/3M-VHB-Tape-4910-Length/dp/B00GTABM3Y)
+- [3D-Printed mounts](https://github.com/PX4/PX4-user_guide/raw/main/assets/airframes/vtol/omp_hobby_zmo_fpv/OMP-Hobby-ZMO-3D-Prints.zip)
+  - 2x wing connector mount
+  - 1x Airspeed sensor mount
+  - 1x GPS-Mount
+  - 1x Lidar-Mount
+  - 1x Skynode mount
+- [USB camera (included in Skynode dev kit)](https://www.amazon.com/ELP-megapixel-surveillance-machine-monitor/dp/B015FIKTZC)
+- Screws, inserts, heat shrink, etc.
+
+## Tools
+
+The following tools were used for this build.
+
+- Hex driver set
+- Wrench set
+- Soldering station
+- Glue: Hot glue, 5 min Epoxy
+- Tape
+- 3M Double sided tape ([3M VHB tape](https://www.amazon.in/3M-VHB-Tape-4910-Length/dp/B00GTABM3Y))
+- Sandpaper
+- 3D-Printer
+
+## Hardware Integration
+
+### Preparations
+
+Remove the original flight controller, ESC and wing connector cables.
+Also remove the the propellers.
+This will help you with the handling of the vehicle and will reduce the risk of an injury due to an unintentional motor startup.
+
+ZMO FPV in it's original state.
+
+![ZMO FPV in it's original state](../../assets/airframes/vtol/omp_hobby_zmo_fpv/zmo-01.jpg)
+
+Flight controller and wing connectors removed from the vehicle.
+
+![ZMO FPV with FC and wing connectors removed](../../assets/airframes/vtol/omp_hobby_zmo_fpv/zmo-02.jpg)
+
+### ESCs
+
+1. Unsolder the ESC PWM-signal and ground pins and solder some servo extension wire to the pins.

```

Comment:
FYI @tizian07 All the way through you will note I have modified this to a list. I prefer this because otherwise the instructions can be a "wall of text". By putting in list you can check that the instructions make sense in order, and put the images associated with each step where it is relevant.

It isn't "always better" but it can make these things easier to follow.